app/2.0/docs/tools/glossary.md

```diff
@@ -0,0 +1,160 @@
+
+### `polymer build`
+This command builds your project with a preconfigured build pipeline. It calls `polymer-build` with options configured for a build suitable for an [app shell](#app-shell) app.
+
+To customize your build pipeline, you can use `polymer-build` itself in conjunction with other build tools.
+
+### `polymer init`
+Initializes a Polymer project. Contains a set of starter templates for elements and apps. 
+
+### `polymer install`
+Install command for Polymer apps. Installs [bower](#bower) dependencies, equivalent to running `bower install`. With the option `--variants`, also installs any [dependency variants](#dependency-variants) specified in the `variants` property of [`bower.json`](#bower-json).
+
+### `polymer lint`
+Invokes [`polymer-linter`](#polymer-linter) to [lint](#linter) a polymer project.
+
+### `polymer serve`
+This command starts [`polyserve`](#polyserve), a simple development server for Web Components.
+
+### `polymer test`
+Runs tests for your Polymer app with [`web-component-tester`](#web-component-tester).
+
+## polymer-build
+`polymer-build` is a library for building Polymer projects. `polymer-build` allows you to completely customize your build and combine additional streams and build tasks in any order. 
+
+Compare the `polymer build` command, which calls `polymer-build` with preconfigured options. 
+
+## polymer-bundler
+Polymer build tool that follows HTML Imports and `<script>` tags to concatenate resources into a single page for production. Can improve performance by reducing network trips.
+
+## polymer-linter
+[Linter](#linter) invoked by `polymer lint` command. Lints a Polymer project.
+
+## polymer-analyzer
+A static analysis framework for web applications. 
+
+## `polymer.json`
+`polymer.json` is the Polymer build configuration file. It lets you configure your app’s entrypoint and app shell files, instead of passing these as parameters to polymer build.
+
+`polymer.json` also lets you specify the locations of source files, HTML fragments and additional dependencies, and configure the [Polymer linter](#polymer-linter).
+
+## `polyserve`
+`polyserve` is a locally-installable web server for developing [bower](#bower) components. 
+
+# R
+## routing
+Routing is the process an app uses to determine how to respond to a client request for a particular endpoint. A default route is the endpoint the app directs the user to when no route is specified. In a PWA, the [app shell](#app-shell) performs routing.
+
+# S
+## service worker
+A script that a web browser runs in the background. On first accessing a particular site, the service worker fetches a defined set of resources and caches them. Then, the service worker acts as a client-side proxy. It intercepts network requests for resources and, where possible, supplies the resources from the browser cache. 
```

Comment:
Thx, I've updated to use this definition. I think the stuff about helping to make apps usable offline is important so I have left it in there as an example. Lmk if it doesn't make sense.